### DISCUSSION TITLE
Kindly help me in fixing this problem.     [ module 'supervision' has no attribute 'Position' ]

### DISCUSSION THREAD

**Discussion by Abhijnan-Maji** [roboflow/supervision#486] [General]
def initiate_polygon_zones(
    polygons: List[np.ndarray],
    frame_resolution_wh: Tuple[int, int],
    triggering_position: sv.Position = sv.Position.CENTER
) -> List[sv.Polygonzone]:
    return [
        sv.Polygonzone(
            polygon=polygon,
            frame_resolution_wh=frame_resolution_wh,
            triggering_position=triggering_position,
        )
        for polygon in polygons
    ]

error:

module 'supervision' has no attribute 'Position'

**Comment by SOUMIA21** [comment]
Any solution to this issue
